### ISSUE TITLE
An auto-eat module

### ISSUE THREAD

**Issue by slyafwalrus** [AMereBagatelle/AFKPeace#18]
You should be able to put food in you offhand or main hand, and if your hunger dips below a certain level, the mod will make the player eat.

**Comment by TheAlexbrine** [comment]
Agreed. As a farm-dependent player, I would have some peace of mind knowing that I'm not at half a heart all night. It's not fatal, but uncomfortable.
This comment seems old, so I'm just voicing my support for this but don't expect for it to be added.
